templates/detailed.tpl

```diff
@@ -15,4 +15,6 @@ ${comment.line} Testcase Example:  ${testcase}
 ${comment.line}
 {{ desc.forEach(function(x) { }}${comment.line} ${x}
 {{ }) }}${comment.end}
+${comment.singleLine} @lc code=start
```

Comment:
I'm thinking that adding a single line between the block comments and the single line comments. Because in some of the languages the comments styles are different.

Besides they belongs to different section.

What do you think?

Reply:
Yes, adding a single empty line is better.